### DISCUSSION TITLE
Configuring Redisson Client for AWS ElastiCache: Endpoint vs. Node List

### DISCUSSION THREAD

**Discussion by sachinbhapkar** [redisson/redisson#6226] [Q&A] [answered]
When setting up the Redisson client for an AWS ElastiCache cluster, should we provide the configuration endpoint or a list of nodes?
ref - https://docs.aws.amazon.com/AmazonElastiCache/latest/dg/Endpoints.html

**Comment by mrniko** [comment, marked as answer]
The configuration endpoint is enought

**Reply by sachinbhapkar** [reply]
> The configuration endpoint is enought

ok thanks, but we are currently using the node addresses, should we change it to the configuration endpoint?

**Reply by mrniko** [reply]
Yes, it's recommended by AWS
